changelog.d/645.change.rst

```diff
@@ -0,0 +1,5 @@
+It is now possible to specify hooks that are called whenever an attribute is set **after** a class has been instantiated.
```

Comment:
Is this file necessary?

Reply:
Unfortunately that's the only way to tell towncrier to link to another issue.